typhos/utils.py

```diff
@@ -420,6 +420,15 @@ def no_device_lazy_load():
         Device.lazy_wait_for_connection = old_val
 
 
+def pyqt_class_from_enum(enum):
```

Comment:
Wouldn't it be best to move this into `pcdsutils` at the `qt` module?

Reply:
sure - once the PR is finished we can move it

Reply:
Nein... move it now! :neckbeard:

Reply:
Move pydm to mily

Reply:
Move your 🍑 to ...